shopfloor_mobile/static/wms/src/css/main.css

```diff
@@ -71,3 +71,10 @@ I tested only w/ checkout/select_package for now
     top: 5px;
     right: 5px;
 }
+
+/*
+Split the lines correctly in the picking note if there are multiple lines
+*/
+.picking-note {
```

Comment:
instead of using a specific class, define a generic one that you can reuse anywhere no matter what you need to render.

Eg: `split-text-lines` or similar.